server/backend/cron/feedback_email.go

```diff
@@ -0,0 +1,139 @@
+package cron
+
+import (
+	"bytes"
+	"crypto/tls"
+	htmlTemplate "html/template"
+	"os"
+	"strconv"
+	textTemplate "text/template"
+	"time"
+
+	"github.com/TUM-Dev/Campus-Backend/server/model"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/gomail.v2"
+
+	_ "embed"
+)
+
+// iterate is a template helper to make counting possible
+func iterate(count int32) []int32 {
+	var items []int32
+	var i int32
+	for i = 0; i < count; i++ {
+		items = append(items, i)
+	}
+	return items
+}
+
+//go:embed email_templates/feedback_body.gohtml
+var htmlFeedbackBody string
+
+//go:embed email_templates/feedback_body.txt.tmpl
+var txtFeedbackBody string
```

Comment:
Probably want to have these above the first function at the top of the file after imports